webapp/components/popover_list_members.jsx

```diff
@@ -57,7 +57,11 @@ export default class PopoverListMembers extends React.Component {
         const members = this.props.members;
         const teamMembers = UserStore.getProfilesUsernameMap();
         const currentUserId = UserStore.getCurrentId();
-        const ch = ChannelStore.getCurrent();
+        const ch = ChannelStore.get(this.props.channelId);
+
+        if (!ch) {
```

Comment:
You need to listen for channel store updates or it will never try to re-render after a failure.